src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -370,6 +372,26 @@ public void afterBulk(long executionId, BulkRequest request, Throwable failure)
     };
   }
 
+  /**
+   * Returns the formatted error message based on customers need to
+   * log exception traces.
+   * @param response The BulkItemResponse returned from Elasticsearch
+   * @param logSensitiveData Boolean flag to identify if customer needs to
+   *                         log exception traces for debugging
+   * @return String Formatted error message
+   */
+  private String getErrorMessage(BulkItemResponse response, boolean logSensitiveData) {
+    if (logSensitiveData) {
+      return response.getFailureMessage();
+    }
+    return String.format("Response status: '%s',\n"
+            + "Index: '%s',\n Document Id: '%s'. \n"
+            + "Ignoring and will not index record.",
```

Comment:
I think, this need not be part of this generic error message
```Ignoring and will not index record.```

Reply:
Thanks @subhashiyer9 for pointing out. Made this change